packages/chakra-ui-core/src/components/theme-editor/ThemeEditor.tsx

```diff
@@ -11,10 +11,17 @@ type ThemeEditorChild = React.ReactElement<
 
 export type ThemeEditorProps = {
   children: ThemeEditorChild[] | ThemeEditorChild
+  isThemeEditorDrawerOpen?: boolean
 }
 
-export const ThemeEditor: FC<ThemeEditorProps> = ({ children }) => {
-  const { isOpen, onOpen, onClose } = useDisclosure()
+export const ThemeEditor: FC<ThemeEditorProps> = ({ children, isThemeEditorDrawerOpen }) => {
+  const [isOpen, setIsOpen] = useState(isThemeEditorDrawerOpen)
```

Comment:
We should not re-implement functionalities that are already available from Chakra UI.
I think we can just use the `useDisclosure()` hook and pass to it the `isOpen`, `onClose`, etc. props.